packages/assert/src/index.ts

```diff
@@ -41,8 +41,8 @@ export const assert = {
         value: string,
         stringEnum: any /* There is no base type for every string enum */,
     ): void {
-        const doesBelongToStringEnum = !_.isUndefined(stringEnum[value]);
-        const enumValues = _.keys(stringEnum);
+        const enumValues = _.values(stringEnum);
+        const doesBelongToStringEnum = _.includes(enumValues, value);
```

Comment:
Our assertion was previously testing the enum keys, not values (which is fine when they are the same... but not when they are not).